minlzdec/CMakeLists.txt

```diff
@@ -10,6 +10,6 @@ if(MSVC)
     set(CMAKE_C_FLAGS_RELWITHDEBINFO "/Ox /Ob2 /Oi /Ot /Oy /GF /Gy /MT /Zi /permissive-")
     set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO "/OPT:ICF /OPT:REF /DEBUG /EMITPOGOPHASEINFO /NOVCFEATURE /NOCOFFGRPINFO /FILEALIGN:512 /DRIVER /MANIFEST:NO /PDBALTPATH:minlzdec.pdb")
 else()
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wno-multichar")
-    set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS} -Ofast -Wall -Werror")
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wconversion -Wno-sign-conversion -Wno-multichar")
```

Comment:
Can you fix the spacing? See the diff. Thanks!

Reply:
Amended commit to fix.